ts/consts.ts

```diff
@@ -2,12 +2,39 @@
 
 import { PathOptions } from 'leaflet';
 
+/**
+ * Base64 data-url for a transparent 1px png image
+ */
 export const TRANSPARENT_PIXEL: string = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkYAAAAAYAAjCB0C8AAAAASUVORK5CYII=';
+/**
+ * Default value for delays caused by animations (in milliseconds)
+ */
 export const ANIMATION_DELAY: number = 50;
+/**
+ * Some lorem ipsum test content
+ */
 export const EXAMPLE_CONTENT: string = 'Vel ipsum odit quia velit omnis illo voluptatem ut. Aperiam porro voluptates maiores.';
+/**
+ * URL-schema of the official osm tile-server
+ */
 export const OSM_TILE_LAYER_URL: string = 'http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png';
+/**
+ * URL for an example image for image-overlays
+ */
 export const IMAGE_OVERLAY_URL: string = 'http://live.osgeo.org/_images/osgeolive_menu6.png';
```

Comment:
URL not working ....

Reply:
That is not really a problem. We just use it in the unit tests to see that the urls get handled correctly. But we can provide another / better one.

Do you have one?

Reply:
@atd-schubert 
I think that was the pic: [desktop10_osmF32.png](http://download.osgeo.org/livedvd/10.0/desktop10_osmF32.png) (also from osgeo)
or here: [osgeolive_menu.png](https://3.bp.blogspot.com/-GY_wstlz9VE/V6jzgzyatGI/AAAAAAAAA2c/LmIvkvtFNfwXM2Yu6nFWY6Hh_wRNHAhvgCLcB/s1600/osgeolive_menu.png) (not a good source)